Fix rowSelection columnWidth  doesn't work

Go7hic opened the pull request:
Fix the Table rowSelection columnWidth  doesn't work
修复 Table 组件自定义列表选择框宽度无效的情况

First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your pull request, thank you!

* [ ] Make sure that you propose pull request to right branch: bugfix for `master`, feature for branch `feature`.
* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [ ] Rebase before creating a pull request to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you pull request.

Extra checklist:

**if** *isBugFix* **:**

  * [x] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

Go7hic committed:
Fix rowSelection columnWidth  doesn't work

Fix the Table rowSelection columnWidth  doesn't work
修复 Table 组件自定义列表选择框宽度无效的情况

afc163 commented:
It will be default selection column exordinary large:

<img width="886" alt="image" src="https://user-images.githubusercontent.com/507615/50582180-66dd2700-0e9b-11e9-95e0-653289ab958b.png">

dyygtfx commented:
Instead of  write the css, i think we can add a defaultProps in rowSelection

Go7hic committed:
Add the Fix code of rowSelection columnWidth

I have add the fix code of rowSelection columnWidth, pls review it. maybe you have a better idea

afc163 commented:
`npm test -- -u` to update test snapshots.

afc163 commented:
@dyygtfx

afc163 commented:
#14115
